.github/workflows/run-stream-processing-offload.yml

```diff
@@ -0,0 +1,100 @@
+name: Stream Processing Offload (HAProxy) tests
```

Comment:
Would it be possible to put this inside run-end-to-end-yml ? for many reason in downstream processes, it'll be easier to maintain if you do not multiply those yml files